makeservices/easywp

```diff
@@ -69,4 +69,30 @@ wp core download
r";b:0;s:34:"/wp-site-health/v1/directory-sizes";b:0;}}s:6:"editor";a:2:{s:13:"default_allow";b:0;s:10:"allow_list";a:62:{s:1:"/";b:0;s:9:"/batch/v1";b:0;s:11:"/oembed/1.0";b:0;s:17:"/oembed/1.0/embed";b:0;s:17:"/oembed/1.0/proxy";b:0;s:6:"/wp/v2";b:0;s:12:"/wp/v2/posts";b:0;s:32:"/wp/v2/posts/(?P&lt;id&gt;[\d]+)";b:0;s:46:"/wp/v2/posts/(?P&lt;parent&gt;[\d]+)/revisions";b:0;s:66:"/wp/v2/posts/(?P&lt;parent&gt;[\d]+)/revisions/(?P&lt;id&gt;[\d]+)";b:0;s:42:"/wp/v2/posts/(?P&lt;id&gt;[\d]+)/autosaves";b:0;s:66:"/wp/v2/posts/(?P&lt;parent&gt;[\d]+)/autosaves/(?P&lt;id&gt;[\d]+)";b:0;s:12:"/wp/v2/pages";b:0;s:32:"/wp/v2/pages/(?P&lt;id&gt;[\d]+)";b:0;s:46:"/wp/v2/pages/(?P&lt;parent&gt;[\d]+)/revisions";b:0;s:66:"/wp/v2/pages/(?P&lt;parent&gt;[\d]+)/revisions/(?P&lt;id&gt;[\d]+)";b:0;s:42:"/wp/v2/pages/(?P&lt;id&gt;[\d]+)/autosaves";b:0;s:66:"/wp/v2/pages/(?P&lt;parent&gt;[\d]+)/autosaves/(?P&lt;id&gt;[\d]+)";b:0;s:12:"/wp/v2/media";b:0;s:32:"/wp/v2/media/(?P&lt;id&gt;[\d]+)";b:0;s:45:"/wp/v2/media/(?P&lt;id&gt;[\d]+)/post-process";b:0;s:37:"/wp/v2/media/(?P&lt;id&gt;[\d]+)/edit";b:0;s:13:"/wp/v2/blocks";b:0;s:33:"/wp/v2/blocks/(?P&lt;id&gt;[\d]+)";b:0;s:43:"/wp/v2/blocks/(?P&lt;id&gt;[\d]+)/autosaves";b:0;s:67:"/wp/v2/blocks/(?P&lt;parent&gt;[\d]+)/autosaves/(?P&lt;id&gt;[\d]+)";b:0;s:12:"/wp/v2/types";b:0;s:35:"/wp/v2/types/(?P&lt;type&gt;[\w-]+)";b:0;s:15:"/wp/v2/statuses";b:0;s:40:"/wp/v2/statuses/(?P&lt;status&gt;[\w-]+)";b:0;s:17:"/wp/v2/taxonomies";b:0;s:44:"/wp/v2/taxonomies/(?P&lt;taxonomy&gt;[\w-]+)";b:0;s:17:"/wp/v2/categories";b:0;s:37:"/wp/v2/categories/(?P&lt;id&gt;[\d]+)";b:0;s:11:"/wp/v2/tags";b:0;s:31:"/wp/v2/tags/(?P&lt;id&gt;[\d]+)";b:0;s:12:"/wp/v2/users";b:0;s:32:"/wp/v2/users/(?P&lt;id&gt;[\d]+)";b:0;s:15:"/wp/v2/users/me";b:0;s:66:"/wp/v2/users/(?P&lt;user_id&gt;(?:[\d]+|me))/application-passwords";b:0;s:77:"/wp/v2/users/(?P&lt;user_id&gt;(?:[\d]+|me))/application-passwords/introspect";b:0;s:90:"/wp/v2/users/(?P&lt;user_id&gt;(?:[\d]+|me))/application-passwords/(?P&lt;uuid&gt;[\w\-]+)";b:0;s:15:"/wp/v2/comments";b:0;s:35:"/wp/v2/comments/(?P&lt;id&gt;[\d]+)";b:0;s:13:"/wp/v2/search";b:0;s:59:"/wp/v2/block-renderer/(?P&lt;name&gt;[a-z0-9-]+/[a-z0-9-]+)";b:0;s:18:"/wp/v2/block-types";b:0;s:54:"/wp/v2/block-types/(?P&lt;namespace&gt;[a-zA-Z0-9_-]+)";b:0;s:85:"/wp/v2/block-types/(?P&lt;namespace&gt;[a-zA-Z0-9_-]+)/(?P&lt;name&gt;[a-zA-Z0-9_-]+)";b:0;s:15:"/wp/v2/settings";b:0;s:13:"/wp/v2/themes";b:0;s:42:"/wp/v2/themes/(?P&lt;stylesheet&gt;[\w-]+)";b:0;s:14:"/wp/v2/plugins";b:0;s:54:"/wp/v2/plugins/(?P&lt;plugin&gt;[^.\/]+(?:\/[^.\/]+)?)";b:0;s:29:"/wp/v2/block-directory/search";b:0;s:18:"/wp-site-health/v1";b:0;s:43:"/wp-site-health/v1/tests/background-updates";b:0;s:42:"/wp-site-health/v1/tests/loopback-requests";b:0;s:37:"/wp-site-health/v1/tests/https-status";b:0;s:45:"/wp-site-health/v1/tests/dotorg-communication";b:0;s:45:"/wp-site-health/v1/tests/authorization-header";b:0;s:34:"/wp-site-health/v1/directory-sizes";b:0;}}}}'
```

Comment:
This seems pretty much unmaintainable. Could you please leave a comment about what this does?

Also is it at all possible to put these settings in some file somewhere?

Reply:
Absolutely, this is very distant but I still recall...
This is the settings for `disable-json-api` to work with its maximum potential.
To get it: first, I install `disable-json-api`, then I change its settings so that they fit our needs. I then check what is in `wp_options.disable_rest_api_options`, where settings are hopefully stored. I copy it out then paste it here.

Reply:
Yeah perhaps reading from a file into the variable would be a bit cleaner. But I guess the settings are just in an unfortunate format.

Reply:
the real issue is this does not work...

Reply:
Ah, yeah that is a problem...

Reply:
BTW I don't quite recall what's the context, maybe check here https://github.com/ocf/utils/issues/168 and see if configuring for users are necessary (I remember this only adds extra functions, without configuring there's still some REST APIs disabled)

Reply:
Yeah the main issue was that security contacted us and informed us that the REST API by default allows user enumeration so we had to tell a couple of vhosts to install the plugin. The hope is that if we install it by default we won't need to deal with a bunch of security emails.

Reply:
Oh and I think the issue with authenticated vs unauthenticated access was we were worried that it would break plugins if we installed it.

Perhaps for new installations that isn't an issue if we tell people they can allow access for authenticated requests?